examples/tracing/golang-push/README.md

```diff
@@ -0,0 +1,14 @@
+## Golang Example
```

Comment:
I like this concise README.md. However, I followed the approach of writing a more detailed (likely unnecessary) one following java's approach.

I think we should make all READMEs in the `/tracing` directory consistent. I can update both my example and the Java one to follow this simpler approach, wdyt?

Reply:
Fully agree!

Though perhaps this is a bit too short and we want to explain what this example is for.

Reply:
+1

Reply:
Yeah I was not planning to keep it like that, but I also did't necessary like the other one 😂  Fill follow up with a PR